vdev/src/commands/release/push.rs

```diff
@@ -0,0 +1,34 @@
+use std::env;
+
+use anyhow::Result;
+use clap::Args;
+
+use crate::util;
+use crate::git;
+
+/// Pushes new versions produced by `make release` to the repository
+#[derive(Args, Debug)]
+#[command()]
+pub struct Cli {}
+
+impl Cli {
+    pub fn exec(self) -> Result<()> {
+        let version = env::var("VECTOR_VERSION").or_else(|_| util::read_version())?;
```

Comment:
Since the original is calling `vdev version` to get its value, I think it would be best to move the guts of `commands/version.rs` into a function and call that here.

Reply:
To clarify, would you like me to add a function inside another module like util::read_version()? Or add a get_version() function inside `release/push.rs`?

Reply:
The former. Putting shared functions inside the individual commands is a poor pattern.

Reply:
Gotcha, what should this module be named? Should it exist in the app one? So it'll be called as app::version()? I will implement this under app for now and await your review for the other changes as well.